### PR TITLE
trying to filter out blank images

### PR DESCRIPTION
**Pull request by martinraison**
@eishay @stkem @andrewconner After looking a bit, it turns out there are interesting images that are under 1.5kb. However the problem that both me and Eishay had was with images named "blank.jpg". So for now I'm just filtering out images that are named "blank.jpg" or "blank.png" - I think we'll have fewer false negatives this way.

### CODE REVIEW

File: kifi-backend/modules/scraper/app/com/keepit/commanders/ScraperURISummaryCommander.scala [hunk 1]
```diff
@@ -114,3 +118,17 @@ class ScraperURISummaryCommanderImpl @Inject()(
...
 
 }
+
+object ScraperURISummaryCommander {
+
+  def filterImageByUrl(url: String): Boolean = {
+    URI.parse(url) match {
+      case Success(imageUri) => {
+        imageUri.path map { path =>
+          !path.endsWith("/blank.jpg") && !path.endsWith("/blank.png")
```
Comment (eishay) [inline kifi-backend/modules/scraper/app/com/keepit/commanders/ScraperURISummaryCommander.scala]: what about other formats like gif?

Comment (eishay) [inline kifi-backend/modules/scraper/app/com/keepit/commanders/ScraperURISummaryCommander.scala]: case sensitive ? how about `blank.JPG`?

**Comment by eishay** [comment]
good idea!

**Comment by eishay** [comment]
would `background.jpg` apply as well?

**Comment by andrewconner** [comment]
Any way to do it based on file size too? <2kb, ignore?

**Comment by andrewconner** [comment]
Sorry just saw "After looking a bit, it turns out there are interesting images that are under 1.5kb". Example?

File: kifi-backend/modules/scraper/app/com/keepit/commanders/ScraperURISummaryCommander.scala (hunk 1 above)
Comment (martinraison) [inline kifi-backend/modules/scraper/app/com/keepit/commanders/ScraperURISummaryCommander.scala]: just updated to be case insensitive and support *.gif.

**Comment by martinraison** [comment]
I'm wary about adding "background.jpg" images because of sites like this one http://deareuropewearesorry.eu/ (the image is not actually called background.jpg but it could)

**Comment by martinraison** [comment]
@andrewconner 
juuzp

The second one is 3kb, but you get the idea

**Pull request merged**